plonky2/src/plonk/verifier.rs

```diff
@@ -12,7 +12,7 @@ use crate::plonk::validate_shape::validate_proof_with_pis_shape;
 use crate::plonk::vanishing_poly::eval_vanishing_poly;
 use crate::plonk::vars::EvaluationVars;
 
-pub(crate) fn verify<F: RichField + Extendable<D>, C: GenericConfig<D, F = F>, const D: usize>(
+pub fn verify<F: RichField + Extendable<D>, C: GenericConfig<D, F = F>, const D: usize>(
```

Comment:
Usually we go through `CircuitData::verify`, was there an issue with that?

Reply:
Oh oops! Fixed